manifests/params.pp

```diff
@@ -40,10 +43,12 @@
       $provider = 'puppet_gem'
       $confdir  = '/etc/puppetlabs/code'
       $cmdpath  = ['/opt/puppetlabs/puppet/bin', '/usr/bin', '/usr/local/bin']
+      $manage_package = false
     } else {
-      $provider = 'gem'
-      $confdir  = '/etc/puppet'
-      $cmdpath  = ['/usr/bin', '/usr/local/bin']
+      $provider       = 'gem'
+      $confdir        = '/etc/puppet'
+      $cmdpath        = ['/usr/bin', '/usr/local/bin']
+      $manage_package = true
```

Comment:
Hmm. Would defaulting to managing the package be considered backwards incompatible? It might cause resource conflicts if any foss user is already managing the package.